source/documentation/troubleshooting/index.md

```diff
@@ -72,6 +72,12 @@ filter {
         }
     }
 
+    # Cloud Foundry passes the app name, space and organisation in the syslog_host
+    # Filtering them into separate fields makes it easier to query multiple apps in a single Kibana instance
+    dissect {
+        mapping => { "syslog_host" => "%{[cf][org]}.%{[cf][space]}.%{[cf][app]}" }
```

Comment:
hmm, is it worth specifying a [`tag_on_failure`](https://www.elastic.co/guide/en/logstash/current/plugins-filters-dissect.html#plugins-filters-dissect-tag_on_failure)?  By default this will tag `_dissectfailure`, but I've found it useful in the past to have a unique tag per filter to make it clear where in the filter configuration the failure happened.

Of course, right now this is the only `dissect` filter in the config, but that may well change.

I propose something like `tag_on_failure => ["_sysloghostdissectfailure"]` although naming is hard.

Reply:
@philandstuff @davbo I'm going to make Phil's proposed change and then get this merged